core/modules/wbase/Base.cc

```diff
@@ -373,4 +319,4 @@ void StringBuffer2::_setSize(unsigned size) {
     _bufferSize = size;
 }
 
-}}} // namespace lsst::qserv::wbase
+}}} // namespace
```

Comment:
Main issue for me is that I like to know it's the end of a namespace instead of a class or function. It's easy for the actual names listed in the comment to fall out of sync with the actual names, and the comment duplicates what's in the code instead of explaining the code.